style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
Here you are using the tags, My suggestion is to use a class attribute for best practices, try to change it and in the future, it'd be really useful to apply DRY (Don't Repeat Yourself ) method.

Reply:
Good! As I have mentioned above this is related to HTML&CSS best practices and it should be a required change.